### ISSUE TITLE
403 Client Error when trying to search for users/tweets

### ISSUE THREAD

**Issue by tadavoila** [DocNow/twarc#643]
I'm using twarc for the first time and trying to search/harvest tweets, and I have already configured twarc. When I try to search for something, it doesn't go through, but rather gives me a 403 error and a link. A couple screenshots of what I'm talking about is attached. Could anyone help me out on how to fix this problem?

![image](https://user-images.githubusercontent.com/69366827/173719747-c7e193fe-d9ce-426d-954c-152abd5efd16.png)
![image](https://user-images.githubusercontent.com/69366827/173719863-c04dc2f7-9b3e-4858-bd3f-693f6d1132cb.png)

**Comment by tadavoila** [comment]
> 

Thank you so much! That works!

![image](https://user-images.githubusercontent.com/69366827/173768482-d7bfa0fa-0a1a-429b-91bd-cc40e6bc0b2d.png)

**Comment by edsu** [comment]
So you should also be able to do this now?

    twarc2 search flu > tweets.jsonl

Note if you supply the filename instead of doing stdout redirection you can see a nifty progress bar:

    twarc2 search flu tweets.jsonl

**Comment by tadavoila** [comment]
> twarc2 search flu tweets.jsonl

Yes I am able to do both the search with the ">" and without, and it shows the cool progress bar :)! Thank you!
